pkg/standalone/standalone.go

```diff
@@ -268,15 +278,24 @@ func runZipkin(wg *sync.WaitGroup, errorChan chan<- error, info initInfo) {
 	}
 	args := []string{}
 
+	var imageName string
+
 	if exists {
 		// do not create container again if it exists.
 		args = append(args, "start", zipkinContainerName)
 	} else {
-		imageName := zipkinDockerImageName
-		if info.imageRepositoryURL != "" {
-			imageName = fmt.Sprintf("%s/%s", info.imageRepositoryURL, imageName)
+		if defaultImageRegistryName == githubContainerRegistryName {
+			imageName = zipkinGhcrImageName
+			if info.imageRegistryURL == "" {
+				info.imageRegistryURL = ghcrURL
+			}
+		} else if defaultImageRegistryName == dockerContainerRegistryName {
+			imageName = zipkinDockerImageName
 		}
 
+		if info.imageRegistryURL != "" {
+			imageName = fmt.Sprintf("%s/%s", info.imageRegistryURL, imageName)
```

Comment:
what is expected value here? 
Say `--image-registry` is set to `"testregistry"`
if `DEFAULT_IMAGE_REGISTRY_MODE` is set to ghcr , the final `imageName` is `testregistry/3rdparty/zipkin`
if `DEFAULT_IMAGE_REGISTRY_MODE` is set to dockerhub, the final `imageName` is `testregistry/openzipkin/zipkin`

I say we should default to one value ...

Reply:
that would actually make this simpler. Do you suggest dokcerhub/ghcr for this?